hw/ip/pinmux/data/pinmux.hjson

```diff
@@ -4,6 +4,7 @@
 //
 {
   name:               "pinmux",
+  one_line_desc:      "Multiplexes between on-chip hardware blocks and pins, can be configured at runtime",
```

Comment:
Not sure if one of the following is more appropriate.
Multiplexes between on-chip hardware blocks and pins and can be configured at runtime
Multiplexes between on-chip hardware blocks and pins that can be configured at runtime
Multiplexes between on-chip hardware blocks and pins, which can be configured at runtime

Reply:
Hm, I am not sure what's best. We want to convey that the multiplexing is configurable at run time. The latter two suggestions I read as the blocks and pins are configured at run time.

Reply:
I think adding a ", and" does the job:
> Multiplexes between on-chip hardware blocks and pins, and can be configured at runtime